chia/data_layer/data_store.py

```diff
@@ -317,6 +317,8 @@ async def get_tree_root(self, tree_id: bytes32, *, lock: bool = True) -> Root:
                 "SELECT * FROM root WHERE tree_id == :tree_id AND generation == :generation",
                 {"tree_id": tree_id.hex(), "generation": generation},
             )
+            if cursor.rowcount < 1:
+                raise Exception("tree id not in store")
```

Comment:
Unless the ID can be considered semi-private?

```suggestion
                raise Exception(f"tree id not in store: {tree_id.hex()}")
```